packages/vue3-vite/src/generators/vue3-vite/files/src/app/components/CookiesConsent.spec.ts.template

```diff
@@ -1,29 +1,19 @@
-import { createApp, h } from 'vue';
-import { expect } from '@jest/globals';
+import { mount } from '@vue/test-utils';
```

Comment:
I import `mount` from `@vue/test-utils` instead of making one on our own.

Reply:
👌